File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/filterCodes/FilterCodesImpl.java
package org.ngbp.jsonrpc4jtestharness.rpc.filterCodes;

import org.ngbp.jsonrpc4jtestharness.rpc.filterCodes.model.GetFilterCodes;

import java.util.ArrayList;

public class FilterCodesImpl implements IFilterCodes {
    @Override
    public GetFilterCodes getFilterCodes() {
        GetFilterCodes.Filters filters = new GetFilterCodes.Filters();
        filters.expires = " 10";
        filters.filterCode = 10;
        ArrayList<GetFilterCodes.Filters> filtersList = new ArrayList<>();
        filtersList.add(filters);
        GetFilterCodes getFilterCodes = new GetFilterCodes();
        getFilterCodes.filters = filtersList;
        return getFilterCodes;
    }

    @Override
    public Object setFilterCodes() {
        return null;
    }
}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/filterCodes/FilterCodesImpl.kt
package org.ngbp.jsonrpc4jtestharness.rpc.filterCodes

import org.ngbp.jsonrpc4jtestharness.rpc.filterCodes.model.Filters
import org.ngbp.jsonrpc4jtestharness.rpc.filterCodes.model.GetFilterCodes
import java.util.*

class FilterCodesImpl : IFilterCodes {
    override fun getFilterCodes(): GetFilterCodes? {
        val filters = Filters().apply {
            expires = " 10"
            filterCode = 10
        }
        val filtersList = ArrayList<Filters?>()
        filtersList.add(filters)
        val getFilterCodes = GetFilterCodes()
        getFilterCodes.filters = filtersList
        return getFilterCodes
    }

    override fun setFilterCodes(): Any? {
        return null
    }
}